javascript-vanilla-v2/main.js

```diff
@@ -0,0 +1,158 @@
+// https://docs.flagsmith.com/clients/javascript#initialisation-options
+// code and comments by @gnumoreno and @MatheusLasserre (github ids)
+const debug = 0
+flagsmith.init({
+  // Dev environment key: iefBdPk5jLjp3WfFjiB4MS
+  // Prod environment key: B7PbVXEWTCGhTD3njUxoL3
+  environmentID: 'iefBdPk5jLjp3WfFjiB4MS', // Required, the environment ID for your environment
```

Comment:
What project are these keys for?

Reply:
Javascript-v2 Example in our org. And it follows the same model as every other example in the repository.

Reply:
@gnumoreno